controllers/cryptoController.js

```diff
@@ -0,0 +1,132 @@
+const usersdata = require('../models/crypto')
+/**
+ * Send Money from one User to Other
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+// http://localhost:3000/cryptoTransaction/send
+// content-type: application.json -> hearder
+// JSON body
+// {
+//   "to": "uttam",
+//   "from": "kratika",
+//   "amount": 30,
+//   "currency": "silver"
+// }
+const send = async (req, res) => {
+  try {
+    const request = req.body
+    const [userTo, userFrom, currency, amount] = [request.to, request.from, request.currency, request.amount]
+    const message1 = `${amount} ${currency} Coins Transfered From ${userFrom} to ${userTo}`
+    const verifyUserTO = await usersdata.fetchUser(userTo)
+    const verifyUserFrom = await usersdata.fetchUser(userFrom)
+    if (verifyUserTO && verifyUserFrom) {
+      if (currency === 'brass' && verifyUserFrom.user.coins.brass >= amount) {
```

Comment:
This logic can be abstracted into a different function.

Reply:
ok Working on it